ipapython/ipautil.py

```diff
@@ -1294,15 +1294,17 @@ def unescape_seq(seq, *args):
 
 def escape_seq(seq, *args):
     """
-    escape (prepend '\\') all occurences of sequence in input strings
+    escape (prepend '\\') all occurences of sequence in input strings unless it
+    is already escaped.
```

Comment:
Probably not, I was just thinking about future proofing the code. You are probably right that the function should be as dumb as possible.